docs/source/run.rst

```diff
@@ -20,7 +20,7 @@ gunicorn
 
 Basic usage::
 
-    $ gunicorn [OPTIONS] APP_MODULE
+    $ gunicorn [OPTIONS] [APP_MODULE]
 
 Where ``APP_MODULE`` is of the pattern ``$(MODULE_NAME):$(VARIABLE_NAME)``. The
```

Comment:
Also, it would be nice to mention that this is optional in 19.7+ (perhaps with a versionchanged directive)